src/mozanalysis/historical_targets/desktop.py

```diff
@@ -0,0 +1,80 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from textwrap import dedent
+
+from mozanalysis.metrics import agg_any
+from mozanalysis.historical_targets import TargetDataSource, Target
+
+#: SegmentDataSource: The clients_last_seen table.
+clients_last_seen = TargetDataSource(
```

Comment:
Can we reuse [this](https://github.com/mozilla/mozanalysis/blob/main/src/mozanalysis/segments/desktop.py)? It would be less confusing and more maintainable to not have 2 sources of truth for client segments.

Reply:
I created a new source here because Segments require aggregating SQL functions to work, and I saw possible Targets that would not need them (e.g. locale, language, etc)

Reply:
Could you say more about the case where an aggregation is not needed? For locale, users can change their locale, so I would think something like `agg_any("locale LIKE 'en*'")` would be needed to target all clients that had set their locale to english at any point during a certain time period.

Reply:
Ah, gotcha, that's my unfamiliarity with the data coming through. I'll make the change to reuse the SegmentDataSource objects, thanks!

Reply:
no problem :)

Reply:
Same as above, made the requested change